Contact form

VTY1999 opened the pull request:
**These are changes we have implemented for this project**

- Create a new branch

- Create a linters for html and css

lily-coder reviewed (commented):
Hi @VTY1999 , 

While you made a great effort in this project
![image](https://user-images.githubusercontent.com/63964646/143437168-f99cfa66-84af-4467-80de-20ca88a932f7.png)
, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that the Pull Request is made from the “feature branch” into the `revert-1-feature` branch. Your Pull Request should be made from the “feature branch” into the `main` branch. Please refer to this link [How To Use Github Flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md)
- Good work in capitalizing and making the commit messages descriptive, however, I think you the commit messages should be written in imperative tense, but you have used past tense. Please refer to [Professional repo - basic rules](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) and [Read more about how to write descriptive commit messages.](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8/) for more information.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **Does not count** into the code reviews limit._

Commit:
Image for my project competion

Commit:
Add linters for html and compete project

Commit:
Add linters for css and complete project

Commit:
Made some changes to html

Commit:
Made some changes to css

Commit:
Changes to html

Commit:
Changes to css

Commit:
Images for project

Commit:
New meun bar

Commit:
Changes to html and css

Commit:
Changes to html

Commit:
Changes to css

Commit:
Changes to css

Commit:
Add contact form

steveWDamesJr committed:
Update Contact Form in markup

-Add horizontal rule div below submission button
-Add spacing to placeholder text to reflect figma template

IfzaRasool committed:
Update markup and stylesheet

-Add button type is submit
-Give Style to button
-Give Style to form Id

Commit:
Update on html and stylesheet

-Add margin to form-icon
-Add linters to html and css

HENRYKC24 commented on index.html:
```diff
@@ -28,19 +36,178 @@ <h2>I'am a Software Developer</h2>
+        <li>Terminal</li>
+      </ul>
+     </section> 
+      <footer id="contact-form">
+        <h3 class="info">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h3>
+        <form id="fs-frm" name="simple-contact-form" accept-charset="utf-8" action="https://formspree.io/f/mayvyyqk" method="post">
+            <input type="text" name="name" id="full-name" placeholder="Full name" required maxlength="30">
+            <input type="email" name="_replyto" id="email-address" placeholder="Email address" required>
+            <textarea rows="5" name="message" id="message" placeholder="Enter text here" maxlength="500" required></textarea>
+            <button class="btn4" type="Submit">Get in touch</button>
+        <div class="hr-line"></div>
+          </form>
```
I must commend you for implementing the form element based on all the requirements. Well done:muscle:

HENRYKC24 commented on index.html:
```diff
@@ -7,19 +7,27 @@
 <body>
-
+  <div id="desktop">
+    <nav id="nav-bar">
+        <a class="name">Verissimo Francis</a>
+        <a class="nav-link1">Portfolio</a>
+        <a class="nav-link2">About</a>
+        <a class="nav-link3">Contact</a>
+      <a href="#"><img id="email" src="Images/union1.png" alt="#"></a>
+    </nav>
+    </div>
     
          <header>
-            <a id="logo">My Logo</a><br><br>
+            <a id="logo">Francis</a><br><br>
         <nav class="manu">
-            <img id="img2" src="https://icon-library.com/images/menu-icon-png-3-lines/menu-icon-png-3-lines-10.jpg" alt="#" class="menu">
+            <img id="img2" src="Images/Union.png" alt="#" class="menu">
```
- [ ] While reviewing your code, I noticed that the whole body of your app scrolls from left to right. I really think that this does not give your users a good UX (User Experience). From the [design template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A27), the mobile version is fixed, and only scrolls vertically. **Please, let's try to fix this**.
## Your app
![image](https://user-images.githubusercontent.com/45668340/145202766-14c7864e-e02e-4721-a77a-e57d57c2ab17.png)
![image](https://user-images.githubusercontent.com/45668340/145202949-2389c14c-5a41-4e11-b2c8-3b5bc73bd4b6.png)

HENRYKC24 commented on index.html:
```diff
@@ -28,19 +36,178 @@ <h2>I'am a Software Developer</h2>
+          </form>
+        <ul class="form-icon">
+          
+          <li><a href="#"><img src="Images/Social_Media.png" alt="#"></a></li>
+</ul>
+      </footer>
+  
 <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
-<link href='http://fonts.googleapis.com/css?family=Roboto' rel="stylesheet">
+<link href="http://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
 <link rel="stylesheet" href="style.css">
 </body>
 </html>
```
- [ ] Could you please check what causes an unnecessary space at the button of your mobile version? Sticking to the Figma template and making your app pixel-perfect is a skill that can impress any recruiter inspecting your code/app on GitHub. Please, fix this. I know you can :muscle: 
![image](https://user-images.githubusercontent.com/45668340/145203459-0400c3a8-a984-460e-8090-f0145617e82e.png)

HENRYKC24 commented on index.html:
```diff
@@ -28,19 +36,178 @@ <h2>I'am a Software Developer</h2>
+    <li class="post2">Profesional Art Printing Data</li>
+<li class="support2">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</li>
+</ul>
+<ul class="tagB">
+<li id="tag8">html</li>
+  <li id="tag9">bootstrap</li>
+  <li id="tag10">Ruby</li>
+</ul>
```
- [ ] I think it would be very nice if you add some `margin-top` CSS property to the above:point_up:  `li` element in all the projects in the app. In addition to making your mobile app look good, you need to stick to the design template as strictly as possible.
## Your Design :point_down: 
![image](https://user-images.githubusercontent.com/45668340/145204189-18fb8190-540f-46a3-9180-bb5c89afc11f.png)

## Given Template :point_down: 
![image](https://user-images.githubusercontent.com/45668340/145203820-83eb9797-e02e-4a81-b9de-d3e02f100fa1.png)

HENRYKC24 reviewed (changes requested):
# STATUS: CHANGES REQUESTED :recycle: 
## Hi @VTY1999 :wave:,
![Very Close](https://media.giphy.com/media/nYYtQdLqwwlTe3ccKh/giphy.gif)
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Commit:
Changes made to css

VanessaAoki reviewed (approved):
## STATUS: APPROVED 🟢 
Hi @VTY1999, 

Great job implementing all the changes required by the previous Code Reviewer, your project is complete! There is nothing else to say other than... it's time to merge it! 
Congratulations! 🎉 

<img alt="approved" src="https://64.media.tumblr.com/87e0b6469c957bf896cdec1cab646acf/tumblr_inline_pk1bdakCCp1rpcr9a_500.gif">

### Project highlights:
- [x] All linter checks are passing :heavy_check_mark: 
- [x] Your README file looks very professional and the PR has a descriptive title :memo: 
- [x] Descriptive commits and correct use of the GitHub Flow :fire: 


Cheers and Happy coding!👏👏👏


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
